modules/data.atmosphere/vignettes/tdm_downscaling.Rmd

```diff
@@ -0,0 +1,55 @@
+Temporally Downscale Meteorology
+===============
+
+
+### Subdaily Training Data
+
+Examples:
+
+* Fluxnet2015 30-minute/hourly
+* Ameriflux 30-minute/hourly
+
+### Extract Training Data and Merge All Years Into 1 File 
+
+```{r}
+library(PEcAn.data.atmosphere)
+library(PEcAn.DB)
+download.Fluxnet2015(sitename = 'Niwot Ridge Forest/LTER NWT1 (US-NR1)', username = 'pecan', overwrite = FALSE, outfolder = '~/Example', 
+   start_date = '2000/01/01', end_date = '2014/12/31')
+
+bety     <- list(user = 'bety', password = 'bety', host = 'localhost', dbname = 'bety', driver = 'PostgreSQL', write = TRUE)
+bety$con <- db.open(bety) 
```

Comment:
This code
```
bety <-dplyr::src_postgres(host = bety$host, user = bety$user, password = bety$password, dbname = bety$dbname)
```
has been used in lots of places around PEcAn, though I've asked someone (I'm blanking on who at the moment, to wrap this in a dplyr-compliant function that'd replace PEcAn.DB::db.open (but would be called the same way you're doing right now).